Create Credits.md

5aji opened the pull request:
We can convert it to pretty HTML later.

Commit:
Create Credits.md

shickey commented on Credits.md:
```diff
@@ -0,0 +1,25 @@
+##Credits
+
+Freddy Rice - Original Beartime
```
"Freddy" should be "Freddie"

shickey replied:
Also, we should credit him as "Freddie Rice, '14"

shickey commented on Credits.md:
```diff
@@ -0,0 +1,25 @@
+
+Project Management: Walker Bond '15, Quinn Ellis '15, Carolyn Nye '15, Meghna Kaul '14
+
+### Beartime Web
```
I would suggest leading with information about the current version. Then lower down the page we can credit those who worked on past versions.

shickey commented on Credits.md:
```diff
@@ -0,0 +1,25 @@
+
+### Beartime Web
+
+Beartime was created due to the demand of a modernization and facelift of Bearstatus.
```
"BearTime was created based on demand for a modernized version of BearStatus"

shickey replied:
More fundamental question: is the capitalization "Beartime" or "BearTime"? The UI on the frontend suggests the latter.

Commit:
Update Credits.md

Commit:
fix capitalization

Commit:
more capitalization fixes

Commit:
BearStatus Tense fixes

kincaidoneil committed:
Add miscellaneous changes to CREDITS.md

- Fix spelling and Markdown errors
- Add tables for role/contributors
- Add links to respective GitHub profiles.
- Add miscellaneous verbiage changes

kincaidoneil committed:
Fix hyperlinks in Credits.md

kincaidoneil committed:
Rename Credits.md to CREDITS.md
